### DISCUSSION TITLE
[Proposal]Setup one SIG for operation projects.

### DISCUSSION THREAD

**Discussion by longdafeng** [oceanbase/oceanbase#907] [General]
Right now, there are projects as following:
https://github.com/oceanbase/obdeploy
https://github.com/oceanbase/ob-operator
https://github.com/oceanbase/obagent

These project are all tell user how to maintain OceanBase cluster. so I want to setup one SIG whose name is "operation" to management these projects.  So the SIG can tell us when to release these project and where should these project go. 

For the SIG, I suggest:
@[frf12](https://github.com/frf12) (Chairman)  as PMC
@chris-sun-star PMC
@LINxiansheng as PMC

Any comments are appreciated.

**Comment by caifeizhi** [comment]
I propose SIG members:
@[frf12](https://github.com/frf12) (Chairman) as PMC
@chris-sun-star  PMC
@hnwyllmm as PMC

**Reply by longdafeng** [reply]
Hnwyllmm do much works in dockers, especially how to reduce OB 's memory useage and build the docker image. 

@caifeizhi  Since Hnwyllmm's work can be seen in community, this time is ok for us, but please write down your reason behind your proposal.  Thanks.

**Comment by whhe** [comment]
+1 for listed members.

And I also proposal @namewxt1220 to be a PMC member as he is familiar with golang and k8s, and participated in the development of first version of `obagent`.

**Comment by longdafeng** [comment]
close this proposal, I will raise one new thread for vote.
